apmpackage/apm/changelog.yml

```
@@ -12,6 +12,9 @@
     - description: Remove unused `processor.event` field from logs data streams
       type: enhancement
       link: https://github.com/elastic/apm-server/pull/11494
+    - description: Remove `event.received` fields from logs, metrics and traces data streams
+      type: bugfix
```

```suggestion
      type: enhancement
```

There's no bug in apmpackage (mapping and ingest pipeline) itself

---

thanks, followed up in [cee297b](https://github.com/elastic/apm-server/pull/11602/commits/cee297b83c3485d41e51e86f24c714f89af522fd)